docs/index.md

```diff
@@ -6,172 +6,94 @@ description: Learn about ksqlDB, the event streaming database for creating strea
-line interface (CLI). The ksqlDB CLI acts as a client to ksqlDB
-Server.
-
-## Supported Versions and Interoperability
-
-You can use {{ site.ak }} with compatible {{ site.cp }} and {{ site.aktm }}
-versions.
-
-|    ksqlDB version     | {{ site.ksqldbversion }} |
-| --------------------- | ------------------------ |
-| Apache Kafka version  | 0.11.0 and later         |
-
-## ksqlDB in {{ site.cp }}
-
-ksqlDB is available with {{ site.cp }} on-premises deployments and as a
-fully managed service in {{ site.ccloud }}. For more information, see the
-following topics in the {{ site.cp }} documentation.
-
-- [Install ksqlDB as part of the {{ site.cp }} bundle](https://docs.confluent.io/current/ksqldb/installing.html)
-- [Install](https://docs.confluent.io/current/ksqldb/installing.html), [Operate](https://docs.confluent.io/current/ksqldb/operations.html), and [Upgrade](https://docs.confluent.io/current/ksqldb/upgrading.html) ksqlDB
-- [Run ksqlDB in {{ site.ccloud }}](https://docs.confluent.io/current/cloud/quickstart/ksql.html)
-- [Connect ksqlDB to {{ site.ccloud }}](https://docs.confluent.io/current/cloud/cp-component/ksql-cloud-config.html)
-- [Run ksqlDB in {{ site.c3 }}](https://docs.confluent.io/current/control-center/ksql.html)
-- [Connect ksqlDB to  {{ site.c3 }}](https://docs.confluent.io/current/ksqldb/integrate-ksql-with-confluent-control-center.html)
-- [ksqlDB and role-based access control (RBAC)](https://docs.confluent.io/current/security/rbac/ksql-rbac.html)
-
+## Try it
+
+<div class="cards">
+  <div class="card">
+    <strong>Quickstarts</strong>
+    <ul class="card-items">
+      <li><a href="https://ksqldb.io/quickstart.html">Standalone quickstart</a></li>
+    </ul>
+  </div>
+
+  <div class="card">
+    <strong>How-to guides</strong>
+    <ul class="card-items">
+      <li><a href="/how-to-guides/query-structured-data/">Structured data</a></li>
+      <li><a href="/how-to-guides/use-connector-management/">Connector management</a></li>
+      <li><a href="/how-to-guides/create-a-user-defined-function/">User-defined functions</a></li>
+      <li><a href="/how-to-guides/substitute-variables/">Variables</a></li>
+    </ul>
+    <small><a href="/how-to-guides/" class="card-more">More →</a></small>
+  </div>
+
+  <div class="card">
+    <strong>Tutorials</strong>
+    <ul class="card-items">
+      <li><a href="/tutorials/materialized/">Materialized cache</a></li>
+      <li><a href="/tutorials/etl/">Streaming ETL pipeline</a></li>
+      <li><a href="/tutorials/event-driven-microservice/">Event-driven microservice</a></li>
+    </ul>
+    <small><a href="/tutorials/" class="card-more">More →</a></small>
+  </div>
+</div>
+
+## Learn
+
+<div class="cards">
+  <div class="card">
+    <strong>How it works</strong>
+    <ul class="card-items">
+      <li><a href="https://www.confluent.io/blog/how-real-time-stream-processing-works-with-ksqldb/">The SQL language</a></li>
```

Comment:
I feel like something at this hints more at the overview nature of the blog post. Looking at the text alone, I thought the link was going to be something like the syntax guide.
```suggestion
      <li><a href="https://www.confluent.io/blog/how-real-time-stream-processing-works-with-ksqldb/">Stream processing with SQL</a></li>
```

Reply:
fwiw, the suggestion still fits in the horizontal space
![image](https://user-images.githubusercontent.com/141130/101186068-9e712280-3620-11eb-9d20-f17bb10e662f.png)